lib/openssl/rsassa.c

```diff
@@ -89,14 +89,15 @@ sign(json_t *sig, const json_t *jwk,
     const RSA *rsa = NULL;
     size_t sgl = 0;
     int pad = 0;
+    int slen = 0;
```

Comment:
The declaration of slen needs to be moved up one line. We sort declarations by width.